src/dialects/abstract/index.ts

```diff
@@ -23,6 +23,12 @@ export type DialectSupports = {
   skipLocked: boolean,
   finalTable: boolean,
 
+  /**
+   * Does this dialect support checking `IF NOT EXISTS` before adding column
+   * For instance, in Postgres, "ADD COLUMN IF NOT EXISTS" only adds the column if it does not exist
+   */
+  ifNotExists: boolean,
```

Comment:
Multiple queries support "IF NOT EXISTS", but the support is not uniform across dialects. I recommend you name it like this instead:

```
addColumn: {
  ifNotExists: boolean,
},
```